src/angular/app.js

```diff
@@ -212,8 +212,8 @@ controllers.controller('ResultsCtrl', ['$scope', 'Search', function ($scope, Sea
 
     // gets the activity details of the service 
     $scope.getActivityDetails = function(result){
-        //define a variable to store the 'closing time'
-        var activityDetails = null;
+        // Define a default variable value for activity details
+        var activityDetails = ["Unknown"];
```

Comment:
Sure but if we don't think about it now we're going to build an app that we have no idea how to translate. Then it will be completely useless to our actual users that speak arabic.